distribution.pm

```diff
@@ -114,9 +114,11 @@ Deprecated mode
 
   script_run($program, [$timeout])
 
-Run I<$cmd> (by assuming the console prompt and typing the command). After that, echo
-hashed command to serial line and wait for it in order to detect execution is finished.
-To avoid waiting, use I<$timeout> 0.
+Run I<$cmd> (by assuming the console prompt and typing the command). After
+that, echo hashed command to serial line and wait for it in order to detect
+execution is finished. To avoid waiting, use I<$timeout> 0. If a terminator
+is found, for example '&', C<script_run> tries to produce a (bash) valid
+command line but warns about this discouraged use.
```

Comment:
Judging by the code it looks more like it would thrown an exception (instead of just printing a warning).

Reply:
done